xcode/Subconscious/Shared/Services/Noosphere/Sphere.swift

```diff
@@ -703,8 +703,12 @@ public actor Sphere: SphereProtocol, SpherePublisherProtocol {
             ns_string_array_free(slugs)
         }
         
-        return try slugs.toStringArray().map({ string in
-            try Slug(string).unwrap(SphereError.parseError(string))
+        return try slugs.toStringArray().compactMap({ string in
```

Comment:
`sphere.swift` is meant to be a low-level representation of a sphere. We'll want to put any filtering code like this in an app-specific service instead. That way, we'll still be able to access `_underscored` "machine files" from the lower-level APIs.

Reply:
👍 I've moved this into `UserProfileService` and in the `listRecentMemos` in `DataService`.